src/asset-details/AssetDetailsCard.js

```diff
@@ -33,7 +33,7 @@ export default class AssetDetailsCard extends Component {
 				}
 				<br />
 				<h5><M m="Durations" /></h5>
-				<AssetDetailsDurations durations={durations} />
+				{durations && <AssetDetailsDurations durations={durations} />}
```

Comment:
I think both would work, would you explain your reasoning and maybe we can generalize it as a rule so we have less decision to make =) ?